dev/CommonManaged/MSTestInterop.cs

```diff
@@ -141,10 +141,15 @@ public static void Fail(string message, params object[] args)
             Assert.Fail(message, args);
         }
 
-        public static void Throws<T>(Action action, string message = null) where T : Exception
```

Comment:
What's the difference between having a null default value and adding a overloaded version of that function with only one parameter? 

Does Assert.ThrowsException<T>(action, null) and Assert.ThrowsException<T>(action) behave differently?

Reply:
Yeah, passing null as an argument causes that method to throw invalid argument exception :)